test/test_jit.py

```diff
@@ -1375,6 +1375,8 @@ def func(alpha, beta, x, y) -> (z):
         x = Variable(torch.rand(3).float(), requires_grad=True)
         y = Variable(torch.rand(3).float(), requires_grad=True)
         outputs = alpha * x + beta * y
+        # note - cannot optimize yet because broadcasts are not inserted
```

Comment:
broadcasts?

Reply:
alpha has size 1, and x has size 3, but we don't insert the correct broadcasting logic in the IR. Before we run fusion, we should first insert explicit `expand` nodes.

Reply:
idk why I thought this was about broadcasting as in distributing a tensor over GPUs... Thanks haha